airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -256,6 +256,12 @@ definitions:
       namespace:
         description: Optional Source-defined namespace. Currently only used by JDBC destinations to determine what schema to write to. Airbyte streams from the same sources should have the same namespace.
         type: string
+      available_for_syncing:
+        description: Indicates whether the stream can be synced. A stream might not be syncable because e.g: the input API token doesn't have permissions to sync it. If a stream is unavailable for syncing but is still passed in the configured catalog (i.e: it was requested for syncs) then the connector should ignore it.
+        type: boolean
```

Comment:
As the default case for a stream is "available", perhaps it makes sense for this to be called `unavailable (bool)` to reinforce that `unavailable=false` is the default behavior of a stream.